### ISSUE TITLE
Duplicated output of min/max temperature condition

### ISSUE THREAD

**Issue by domcross** [MycroftAI/skill-weather#70]
Since release 19.02.0 I get strange output: after the weather forecast Mycroft says "min max". After figuring out that dialog-file `min.max.dialog` is missing for my language setting (de-de) now Mycroft gives me duplicated temperature conditions. 

I found commit "[Show high-low-page](https://github.com/MycroftAI/skill-weather/commit/0219b23a1b13f2e2348896d0cf9db4ac4def17b3#diff-743b1adbf37db12e16e01ef62a6f7d72)" where the corresponding changes have been made. Either some additional logic is missing there or not all `*.weather.dialog` files have been changed accordingly.

As far as I can see only `current.local.weather.dialog` has been changed. In total following files must be changed/created:
```
current.local.weather.dialog
current.weather.dialog
forecast.weather.dialog
forecast.local.weather.dialog
hour.weather.dialog
hour.local.weather.dialog
min.max.dialog
```
It should be considered that already eight languages are supported by this skill leading to at least 56 dialog files that must be changed and translated...

**Comment by forslund** [comment]
Thanks for reporting. The duplicate output is handled in the wip branch review-update and will be merged shortly.

**Comment by domcross** [comment]
fixed with commit [8343249](https://github.com/MycroftAI/skill-weather/commit/8343249f89014cb5944f945ba9787bbfb96b46b8#diff-743b1adbf37db12e16e01ef62a6f7d72)
